### ISSUE TITLE
forcePerTimeMeasure - Wrong

### ISSUE THREAD

**Issue by sdeaton75** [DIGGSml/diggs-schema#39]
I believe the forcePerTimeMeasure enumerations are incorrect. These appear to mostly be PressurePerTime.
forcePerTime should be something like a N/s or lbf/sec.

We need to correct this to have both UOM enumerations for PressurePerTime and forcePerTime.  

All enumerations for the current forcePerTime can be moved to PressurePerTime **EXCEPT** **GPa**

**Comment by dponti** [comment]
You're correct. Energistics 2.0 added some pressure/time units so need to harmonize the DIGGS namespace list with the new Energistics list and rename to PressurePerTimeMeasure. There is no current hardcoded property that uses the current ForcePerTimeMeasure (should be PressurePerTimeMeasure) in the current schema, so this is used for test properties at the present time.

Energistics does not list a Force per Time quantity class (eg. lbf/sec) so if we need to consider force per time. Do we have any properties that are of this quantity class? If so, we will need to create them - please provide unit definitions for what might be required (if any are required).

**Comment by sdeaton75** [comment]
We need lbf/sec and N/sec, kN/sec.
